### ISSUE TITLE
GPL-929 - As a laboratory user I would like to hide locations that are inactive (C=S, V=3)

### ISSUE THREAD

**Issue by stevieing** [sanger/labwhere#353]
**User story**
As a laboratory user I would like to hide locations that are inactive. Because we have now fixed the audit history you can no longer delete unused locations. This means that these locations are no longer needed but will appear in the user interface.

**Who are the primary contacts for this story**
Danni

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] When I make a location inactive it should no longer appear in searches or lists
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
duplicate of #473
